package.json

```diff
@@ -17,9 +17,11 @@
   },
   "scripts": {
     "app": "npm run l10n && gulp app",
-    "build": "gulp",
-    "l10n": "pontoon-to-json",
+    "build": "npm run lint && gulp",
```

Comment:
yeah I've taken this task out in one of the follow ups. There... isn't actually anything that uses it! \o/